src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Tree/Dependencies/Snapshot/DependencyTests.cs

```diff
@@ -247,41 +247,38 @@ public void Dependency_SetProperties_PreservesDependencyIDs()
             Assert.Equal(expectedDependencyId, newDependency.DependencyIDs[0]);
         }
 
-        //[Fact]
-        //public void Dependency_DependenciesProperty()
-        //{
-        //    var mockModel1 = IDependencyModelFactory.Implement(
-        //        providerType: "providerType",
-        //        id: "someId1",
-        //        dependencyIDs: new[] { "someId2", "someId_other" });
-
-        //    var mockModel2 = IDependencyModelFactory.Implement(
-        //        providerType: "providerType",
-        //        id: "someId2");
-        //    var mockModel3 = IDependencyModelFactory.Implement(
-        //        providerType: "providerType",
-        //        id: "someId_other");
-
-        //    var targetFramework = ITargetFrameworkFactory.Implement("Tfm1");
-        //    var mockSnapshot = ITargetedDependenciesSnapshotFactory.ImplementMock(targetFramework: targetFramework);
-
-        //    var dependency1 = new Dependency(mockModel1, targetFramework);
-        //    var dependency2 = new Dependency(mockModel2, targetFramework);
-        //    var dependency3 = new Dependency(mockModel3, targetFramework);
-
-        //    var children = new Dictionary<string, IDependency>()
-        //    {
-        //        { dependency2.Id, dependency2 },
-        //        { dependency3.Id, dependency3 },
-        //    };
-
-        //    mockSnapshot.Setup(x => x.DependenciesWorld).Returns(
-        //        ImmutableDictionary<string, IDependency>.Empty.AddRange(children));
-
-        //    Assert.True(dependency1.Dependencies.Count() == 2);
-        //    Assert.True(dependency1.Dependencies.Any(x => x.Equals(dependency2)));
-        //    Assert.True(dependency1.Dependencies.Any(x => x.Equals(dependency3)));
-        //}
+        [Fact]
```

Comment:
I wonder why this was commented out here: https://github.com/dotnet/project-system/commit/c9f4b6c943a04c0b9df1928824327a4951fff263#diff-da666934981933a9cc43c6fdaf1b6d4aL227

Reply:
I can't tell from context. Perhaps @abpiskunov can comment when he's back in the office.